book-library/script.js

```diff
@@ -1,6 +1,6 @@
 let myLibrary = [];
 
-window.addEventListener("load", function (e) {
+window.addEventListener("load", function () {
   populateStorage();
   render();
```

Comment:
You've fixed all the bugs! Nice one! If you've not used it before, we used [Github Codespaces](https://github.com/features/codespaces) to review your work! Good job 🏅